content/posts/07_algorithmic_racism_edj.md

```diff
@@ -0,0 +1,205 @@
+
+One of the challenges I had to pause on in my startup time was, can we, as a handful of people trying to build a product, really afford the extra outreach time to hire for diversity? But can we afford not to? You are in such a rush that you want to cut corners. But every decision you make, **you are deciding what matters for the parts of society you have influence over**.
+
+**KEVIN**: Even EDGI has that problem. Hiring is—it's not that you want it to be fast, but you don't want it to be too slow either, because it drains energy. That becomes an excuse to use shortcuts, like “we don't have time for it”, so we'll just hire the easiest thing. And **the easiest thing is to stay within our networks** and the people that we know already.
+
+**LOURDES**: This whole fast pace thing– this is our life in this society. We are always going, ch-ch-ch-ch—and that's what neoliberalism is about, is this idea of being extra efficient, maximizing your profits. Getting, and always needing, more money.
+
+Sometimes you have to take a step back. I think Coronavirus is helping us do that, helping us say, Wow, there are things that are deeply flawed in the systems that we've created for ourselves, but we also need them. How do we live outside of them now?
+
+**KELSEY**:  And now that people are staying at home—I’m out in the woods; I can go outside. But in the city, maybe you're in an apartment complex, and there’s really no place to go. So people are living online more than ever. But **if everything we do is moderated through a screen and especially through the internet**—who controls the internet? And how?
+
+
+## Accidental encoding of antipatterns
+
+**RICO**: The thing I'm most afraid of is that it's either not a person, not a good person, or not enough people.
+
+I'm going to boil that big idea of “controlling the internet” down into something small, my LinkedIn network. Let's say 25 of my friends get a new job. How do they choose the five to suggest that I congratulate? That's a decision made by an algorithm that encodes some arbitrary set of behaviors. I don’t know what those behaviors are; is it based on the last five people I've messaged, or the five people I haven't messaged for the longest time? If I were to pass on two women in a row, it would then stop recommending that I "Say hello" to women—is that one of the attributes it’s tracking?
+
+**It needs to not just encode just your previous behaviors, because a lot of our previous behaviors are based on the networks that we grew up with**. A lot of us have been locked into these segregated worlds, and* *those worlds are not going to change unless those algorithms are cracked into.**
```

Comment:
```suggestion
**It needs to not just encode just your previous behaviors, because a lot of our previous behaviors are based on the networks that we grew up with**. A lot of us have been locked into these segregated worlds, and **those worlds are not going to change unless those algorithms are cracked into.**
```